tests/spec/date_series_ops/test_date_comparisons.py

```diff
@@ -129,3 +131,26 @@ def test_is_equal_with_str_date(spec_test):
             4: None,
         },
     )
+
+
+def test_is_in_dates(spec_test):
+    dates = [date(2010, 1, 1), date(2010, 1, 2)]
+    spec_test(
+        table_data,
+        p.d1.is_in(dates),
+        {
+            1: False,
+            2: False,
+            3: True,
+            4: None,
+        },
+    )
+
+
+def test_is_in_strings(spec_test):
+    d = ["2010-01-01", "2010-01-02"]
+    spec_test(
+        table_data,
+        p.d1.is_in(d),
+        {1: False, 2: False, 3: True, 4: None},
```

Comment:
If you add a trailing comma after `None` then Black will format these one per line as for the other test cases.